core/iwasm/libraries/lib-socket/inc/wasi_socket_ext.h

```diff
@@ -530,6 +559,198 @@ __wasi_sock_set_send_timeout(__wasi_fd_t fd, uint64_t timeout_us)
             (int32_t)fd, (int64_t)timeout_us);
 }
 
+int32_t
+__imported_wasi_snapshot_preview1_sock_set_keep_alive(int32_t arg0,
+                                                      int32_t arg1)
+    __attribute__((__import_module__("wasi_snapshot_preview1"),
+                   __import_name__("sock_set_keep_alive")));
+
+static inline __wasi_errno_t
+__wasi_sock_set_keep_alive(__wasi_fd_t fd, uint8_t option)
```

Comment:
may I ask about possible values of `option`?

Reply:
Just 0 and 1 like a bool